dlp/api/Snippets.Tests/JobsGetTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using Google.Cloud.Dlp.V2;
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class JobsGetTests : IClassFixture<DlpTestFixture>
+    {
+        private RetryRobot TestRetryRobot { get; } = new RetryRobot();
+        private DlpTestFixture Fixture { get; }
+        public JobsGetTests(DlpTestFixture fixture)
+        {
+            Fixture = fixture;
+        }
+
+        [Fact]
+        public void TestGetDlpJob()
+        {
+            var dlp = DlpServiceClient.Create();
+            var dlpJob = dlp.CreateDlpJob(Fixture.GetTestRiskAnalysisJobRequest());
+
+            TestRetryRobot.ShouldRetry = ex => true;
```

Comment:
I asked to do this - the tests are flaky without the retry robot for jobs, since jobs can take some time to be created. The default settings should be fine, it's what is used elsewhere.